sunpy/io/tests/test_filetools.py

```diff
@@ -60,9 +59,39 @@ def test_read_file_fits_gzip():
         assert np.all(pair[0][0] == np.tile(np.arange(32), (32, 1)).transpose())
 
 
+@pytest.mark.parametrize('fname', ['aia_171_image.fits',
+                                   pathlib.Path('aia_171_image.fits')])
+def test_write_file_fits(fname, tmpdir):
```

Comment:
Could you add some short one line comments to the top of these test functions explaining what they're testing?